explore-analyze/query-filter/languages/esql-rest.md

```diff
@@ -299,6 +299,24 @@ If the response’s `is_running` value is `false`, the query has finished and th
 }
 ```
 
+You can interrupt a running async query and collect the results that have been computed so far by using the [async stop API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-esql-async-query-stop) with the query ID.
```

Comment:
This is GA in 9.0 right, so we don't need additional applies_to tags?

Reply:
Yes, that's correct.